src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -124,20 +124,24 @@ export class StudentProfilePageComponent implements OnInit {
     const modalRef: NgbModalRef = this.ngbModal.open(UploadEditProfilePictureModalComponent);
     modalRef.componentInstance.profilePicLink = this.profilePicLink;
     modalRef.result.then((formData: FormData) => {
-      const paramsMap: { [key: string]: string } = {
-        user: this.user,
-      };
-
-      this.httpRequestService.post('/student/profilePic', paramsMap, formData)
-          .subscribe(() => {
-            this.statusMessageService.showSuccessMessage('Your profile picture has been saved successfully');
-
-            // force reload
-            const timestamp: number = (new Date()).getTime();
-            this.profilePicLink = `${this.backendUrl}/webapi/student/profilePic?${timestamp}`;
-          }, (response: ErrorMessageOutput) => {
-            this.statusMessageService.showErrorMessage(response.error.message);
-          });
+      if (formData) {
```

Comment:
Consider use guard clause.
```javascript
if (!formData) {
    return;
}
```

Reply:
I have updated it, could you have a review?